python/hail/dataset.py

```diff
@@ -1755,24 +1822,33 @@ def filter_variants_expr(self, condition, keep=True):
         >>>                   .filter_variants_expr('va.gene == "CHD8"'))
 
 
-        Remove variants on chromosome 1:
+        Remove all variants on chromosome 1:
 
         >>> vds_filtered = (hc.read('data/example.vds')
         >>>                   .filter_variants_expr('v.contig == "1"',
         >>>                                          keep=False))
 
+        Keep variants on chromosomes 1, 2, and 3:
+
+        >>> vds_filtered = (hc.read('data/example.vds')
+        >>>                   .filter_variants_expr('set("1","2","3").contains(v.contig)')
+
+        Keep variants on chromosomes 1, 2, and 3:
+
+        >>> vds_filtered = (hc.read('data/example.vds')
+        >>>                   .filter_variants_expr('set("1","2","3").contains(v.contig)')
 
         **Notes**
 
-        ``condition`` is in variant context so the following symbols are in scope:
+        ``condition`` is in the variant context so the following symbols are in scope:
 
         - ``v`` (*Variant*): :ref:`variant`
         - ``va``: variant annotations
         - ``global``: global annotations
         - ``gs`` (*Aggregable[Genotype]*): aggregable of :ref:`genotype` for variant ``v``
 
-        For more information, see the documentation on `data representation, annotations <overview.html#>`_, and
-        the `expression language <../expr_lang.html>`_.
+        .. caution::
+        ``va.contig`` is of type String.
```

Comment:
Also this makes no sense to me. Did you mean `v.contig`? Not sure why this is here.